website/docs/d/nat_gateway.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_nat_gateway"
+sidebar_current: "docs-azurerm-datasource-nat-gateway"
+description: |-
+  Gets information about an existing Nat Gateway
+---
+
+# Data Source: azurerm_nat_gateway
+
+Use this data source to access information about an existing Nat Gateway.
+
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Nat Gateway.
+
+* `resource_group_name` - (Required) Specifies the name of the Resource Group where the Nat Gateway should exist.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `location` - Specifies the supported Azure location where the resource should exist.
+
+* `idle_timeout_in_minutes` - Specifies the idle timeout of the Nat Gateway.
+
+* `public_ip_address_ids` - A list of existing `azurerm_public_ip` resource IDs.
+
+* `public_ip_prefix_ids` - A list of existing `azurerm_public_ip_prefix` resource IDs.

```

Comment:
This is worked like a resource, it should read like a data source, what do these IDs mean